reccmp/ghidra_scripts/README.md

```diff
@@ -4,18 +4,28 @@ The scripts in this directory provide additional functionality in Ghidra, e.g. i
 
 ## Setup
 
+There are two ways to install this script:
+- Case 1: You only want to use this script in a decompilation project.
+    - Create a virtual environment inside your _decompilation project_ and install `reccmp` there. Make sure to _not_ install `reccmp` in editable mode (`pip install -e`).
+    - Your decompilation project needs to have a valid `reccmp-build.yml` next to its `reccmp-project.yml`. This is a known limitation.
+- Case 2: You want to actively develop this script.
+    - Create a virtual env in _this repository_ in a top level directory called `.venv` and install this project there (`pip install .` or `pip install -e .`).
+    - Copy `dev_config.example.json` to `dev_config.json` and configure a decompilation target.
+
 ### Ghidrathon
 Since these scripts and its dependencies are written in Python 3, [Ghidrathon](https://github.com/mandiant/Ghidrathon) must be installed first. Follow the instructions and install a recent build (these scripts were tested with Python 3.12 and Ghidrathon v4.0.0).
 
 ### Script Directory
+This step differs slightly depending on your setup.
+
 - In Ghidra, _Open Window -> Script Manager_.
 - Click the _Manage Script Directories_ button on the top right.
-- Click the _Add_ (Plus icon) button and select this file's parent directory.
+- Click the _Add_ (Plus icon) button.
+    - Case 1: Select `Lib/site-packages/reccmp/ghidra_scripts` inside your _project's_ virtual environment.
```

Comment:
Maybe a note here that you have to enable "Show '.' files" under "File Chooser Options" if your environment is under ".venv"